### PR TITLE
64 vector layer in 5514

### PR DESCRIPTION
**Pull request by index-git**
Part of issue #64

- [x] Tests
- ~~Layman Test Client~~
- [x] Changlelog
- [x] Documentation

Optionally
- Add dependency to doc/dependencies.md

### CODE REVIEW

File: src/layman/common/bbox.py [hunk 1]
```diff
@@ -82,15 +82,22 @@ def transform(bbox, crs_from, crs_to):
             min(bbox[2], world_bounds[2]),
             min(bbox[3], world_bounds[3]),
         )
+    params = tuple(bbox)
+    if crs_def.CRSDefinitions[crs_from].proj4text:
+        query_st_trans_params = ', %s, %s'
+        params += (srid_from, crs_def.CRSDefinitions[crs_from].proj4text, srid_to,)
```
Comment (jirik) [inline src/layman/common/bbox.py:88]: Move srid_from to line 85

File: src/layman/common/bbox.py [hunk 2]
```diff
@@ -82,15 +82,22 @@ def transform(bbox, crs_from, crs_to):
...
+    else:
+        query_st_trans_params = ', %s'
+        params += (srid_from, srid_to,)
```
Comment (jirik) [inline src/layman/common/bbox.py:91]: Move srid_from to line 85

File: src/crs/__init__.py [hunk 3]
```diff
@@ -76,4 +76,25 @@
         axes_order_east_north_in_epsg_db=False,
         proj4text=None,
     ),
+    EPSG_5514: CRSTypeDef(
+        world_bbox=(
```
Comment (jirik) [inline src/crs/__init__.py:80]: Use values -951499.37 -1276279.09 -159365.31 -983013.08 (covers also Slovakia)

File: tests/dynamic_data/publications/crs/__init__.py [hunk 4]
```diff
@@ -32,6 +32,20 @@
                           'file_extension': 'shp', },
         KEY_THUMBNAIL: layers.SMALL_LAYER.thumbnail,
     },
+    'epsg_5514': {
+        KEY_FILE_NAME: 'sample_point_cz_5514_5239',
+        KEY_INFO_VALUES: {
```
Comment (jirik) [inline tests/dynamic_data/publications/crs/__init__.py:37]: Publish with existing blue-dot style

File: CHANGELOG.md [hunk 5]
```diff
@@ -17,6 +17,7 @@ make timgen-build
 - [#64](https://github.com/LayerManager/layman/issues/64) Responses of [GET Layers](doc/rest.md#get-layers), [GET Workspace Layers](doc/rest.md#get-workspace-layers), [GET Workspace Layer](doc/rest.md#get-workspace-layer), [PATCH Workspace Layer](doc/rest.md#patch-workspace-layer), [GET Maps](doc/rest.md#get-maps), [GET Workspace Maps](doc/rest.md#get-workspace-maps), [GET Workspace Map](doc/rest.md#get-workspace-map), [PATCH Workspace Map](doc/rest.md#patch-workspace-map) contains new attribute
    - `native_crs` with code of native CRS in form "EPSG:&lt;code&gt;", e.g. "EPSG:4326"
    - `native_bounding_box` with coordinates in native CRS  [minx, miny, maxx, maxy]
+- [#64](https://github.com/LayerManager/layman/issues/64) Layman supports import of vector layers in "EPSG:5514" with sld style.
```
Comment (jirik) [inline CHANGELOG.md:20]: sld -> SLD

**Review by jirik — CHANGES_REQUESTED** [review]

File: src/crs/__init__.py (hunk 3 above)
Comment (index-git) [reply src/crs/__init__.py:80]: You mean, those that are there already? :)

**Review by jirik — APPROVED** [review]

**Pull request merged**